handlers/get.go

```diff
@@ -58,13 +58,18 @@ func (d *Dependency) Get(c *fiber.Ctx) error {
 	// we need to replace escaped newline to literal newline
 	content := strings.Replace(i.Paste, `\n`, "\n", -1)
 
+	c.Set("Content-Type", "text/plain")
 	if qs.Language != "" {
-		highlighted := h.Highlight(content, qs.Language, qs.Theme, qs.LineNr)
+		highlighted, err := h.Highlight(content, qs.Language, qs.Theme, qs.LineNr)
+		if err != nil {
+			// they should still be able to get the plain text even if the highlighter is b0rked
+			c.Status(http.StatusOK).Send([]byte(content))
```

Comment:
if it's error, just `return err`, no need to make a `StatusOK` response

Reply:
kalo highlighternya doang yg error, masa ga ditampilin meskipun ga pake highlight?

Reply:
kemungkinan highlighternya error gara-gara apa?

Reply:
gagal tokenise atau formatter gagal ngehasilin output

Reply:
oke. tetep aja ga ideal. tapi aku benerin nantian